webapp/channels/src/components/size_aware_image.tsx

```diff
@@ -147,11 +170,11 @@ export default class SizeAwareImage extends React.PureComponent {
         }
     };
 
-    handleImageClick = (e) => {
+    handleImageClick = (e: any) => {
```

Comment:
can we remove any from here

Reply:
I am not getting how to resolve that! because one event is coming from keyboard event ! which is used to invoke onclick function ! so i need some help for this!

Reply:
If you see `handleImageClick` is event handler function and these ideally should not be called in another handler function, what i would suggest is to remove calling of this function from another handler function.
```ts
    handleImageClick = (e: React.MouseEvent<HTMLImageElement>) => {
        this.props.onClick?.(e, this.props.src);
    };

    onEnterKeyDown = (e: React.KeyboardEvent<HTMLImageElement>) => {
        if (e.key === 'Enter') {
            this.props.onClick?.(e, this.props.src);
        }
    };
```

Then you would also have to change the original type of onClick to ` onClick?: (e: React.SyntheticEvent<HTMLImageElement | HTMLDivElement>, link?: string) => void;` React.SyntheticEvent type which is a base type for all React events. This type can be used when the specific event type doesn't matter.